api/app/assets/stylesheets/active_admin.scss

```diff
@@ -47,11 +47,6 @@
 //
 //   .status_tag { background: #6090DB; }
 
-.checkbox-label {
-  display: inline;
-}
-
-.checkbox-description {
-  margin-bottom: .5rem;
-  padding-left: 1.5rem;
-}
+form fieldset > ol > li.boolean label {
+  font-weight: bold;
+}
```

Comment:
These seem to have moved into the middle of the form, is there a way to take them back to the left?
![image](https://user-images.githubusercontent.com/45389321/86342720-e47c2180-bc4f-11ea-9342-a7b6002bfd48.png)

Reply:
I think this is an issue with ActiveAdmin for checkboxes on smaller zoom sizes, you can see it here as well: http://demo.activeadmin.info/admin/products/new

I checked this on master, and the current checkbox implementation also gets offset at zoom 90% or lower.